File: Semester 2 (Sipaling Sibuk Tuh, Masokis!)/Tugas/DDP 2/DDP2 - LAB 00 - ALDEN LUTHFI - 2206028932/java/Nilai.java
import java.util.Scanner;


public class Nilai {
    public static void main(String[] args) {
        try (Scanner input = new Scanner(System.in)) {
            
            System.out.print("Masukkan Nama Mahasiswa: ");
            String nama = input.nextLine();
            System.out.print("Masukkan Nilai Asli: ");
            int nilaiAsli = input.nextInt();
            System.out.print("Masukkan Durasi: ");
            int durasi = input.nextInt();

            double indeks;

            if (durasi < 60) {
                indeks = 1.2;
            } else if (durasi <= 70) {
                indeks = 1;
            } else if (durasi < 90) {
                indeks = 0.75;
            } else if (durasi <= 100) {
                indeks = 0.5;
            } else {
                indeks = 0.2;
            }

            System.out.printf("%s mendapatkan nilai akhir: %f\n", nama, nilaiAsli * indeks);
        }
    }
    
}

File: Semester 2 (Sipaling Sibuk Tuh, Masokis!)/Tugas/DDP2/DDP2 - LAB 00 - ALDEN LUTHFI - 2206028932/python/nilai.py
nama = input("Masukkan Nama Mahasiswa: ")
nilai_asli = int(input("Masukkan Nilai Asli: "))
durasi = int(input("Masukkan Durasi: "))

if durasi < 60:
    nilai_akhir = 1.2 * nilai_asli
elif 60 <= durasi <= 70:
    nilai_akhir = nilai_asli
elif 70 < durasi < 90:
    nilai_akhir = 0.75 * nilai_asli
elif 90 <= durasi <= 100:
    nilai_akhir = 0.5 * nilai_asli
else:
    nilai_akhir = 0.2 * nilai_asli

print(nama + " mendapatkan nilai akhir " + str(nilai_akhir))
